app/components/auk/dropdown/item.hbs

```diff
@@ -0,0 +1,27 @@
+<li class={{@class}}>
+  {{#if @href}}
+    <a class="auk-menu__item {{this.skin}}" href={{@href}} ...attributes>
+      {{yield}}
+    </a>
+  {{else if @textOnly}}
+    <div class="auk-menu__item--text-only" ...attributes>
```

Comment:
Ik ben zelf niet 100% overtuigd ivm de `auk-menu__item--text-only` benaming. Als je naar de notatie kijkt zou je denken dat dit een modifier-class is, wat uiteindelijk niet het geval is.

Dit is ok voor nu, maar ik noteer dit even voor mezelf om dit later eens te bekijken.

Reply:
https://github.com/kanselarij-vlaanderen/au-kaleidos-css/issues/11